linux/shim/init.go

```diff
@@ -70,11 +73,14 @@ func newInitProcess(context context.Context, path string, r *shimapi.CreateReque
 		IO:            io,
 		NoPivot:       r.NoPivot,
 	}
+	sigchld := make(chan os.Signal, 2048)
+	// main.go calls Notify as well, but it does not hurt
+	signal.Notify(sigchld, syscall.SIGCHLD)
```

Comment:
ok, and for some reason latest containerd with latest runc no longer hangs up; it properly exit with `rpc error: code = 5 desc = open /run/containerd/linux/foobar-10645/init.pid: no such file or directory` when broken config is specified

let me close this PR